doc/release/1.16.0-notes.rst

```diff
@@ -61,6 +61,12 @@ Now, if the `NPY_DISTUTILS_APPEND_FLAGS` environment variable is set to 1, the
 behavior will be appending.  This applied to: `LDFLAGS`, `F77FLAGS`,
 `F90FLAGS`, `FREEFLAGS`, `FOPT`, `FDEBUG`, and `FFLAGS`.  See gh-11525 for more
 details.
+``np.clip`` and the ``clip`` method check for memory overlap
```

Comment:
Needs empty line (I think) for sphinx to render properly (maybe a rebase artefact?)